frontend/packages/ux-editor/src/hooks/mutations/useUpdateFormComponentMutation.test.ts

```diff
@@ -152,6 +166,58 @@ describe('useUpdateFormComponentMutation', () => {
       }
     }
   });
+
+  describe('Update FileUpload and FileUploadWithTag components', () => {
+    const componentTypes = [ComponentType.FileUpload, ComponentType.FileUploadWithTag];
+
+    componentTypes.forEach((componentType) => {
+      it(`Updates the layout and the bpmn file when updating the id of a ${componentType} component`, async () => {
```

Comment:
Her mener jeg at du kan skrive `it.each(componentTypes)('Updates the layout and the bpmn file when updating the id of a %s component', async (componentType) => {`.